src/main/java/com/epam/spring/homework4/controller/dto/ServiceDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.spring.homework4.controller.dto;
+
+import com.epam.spring.homework4.controller.dto.validation.group.OnCreate;
+import com.epam.spring.homework4.controller.dto.validation.group.OnUpdate;
+import lombok.Builder;
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Null;
+import javax.validation.constraints.Pattern;
+
+@Data
+@Builder
+public class ServiceDTO {
+    int id;
```

Comment:
Hi, I've noticed that almost all fields in DTO's have missing access modifiers (`private`) . Any reason for that?

Reply:
No reasons